test/game-test.js

```diff
@@ -44,7 +44,6 @@ describe('Game', function() {
 
```

Comment:
We are expecting this method to have been called 1 time after startGame is invoked in the domUpdates file. It keeps returning 0.

Reply:
I'm not positive why this is going on... if you run `expect(domUpdates.showPuzzle).to.be.spy`, do you get confirmation that the spy is set?